code/go/0chain.net/core/common/admin.go

```diff
@@ -15,6 +15,7 @@ func SetAdminCredentials() {
 }
 
 func AuthenticateAdmin(handler ReqRespHandlerf) ReqRespHandlerf {
+	SetAdminCredentials()
```

Comment:
Let's call this SetAminiCredentials()  [initHandlers](https://github.com/0chain/blobber/blob/4b3a59abe95e1713de7e75a4dc38fd97a002a262/code/go/0chain.net/blobber/http.go#L86) function, the problem of calling it here is we will update the global variables on every API calls.

Reply:
updated.